docs/tutorials/auth.md

```diff
@@ -0,0 +1,141 @@
+## Basic auth
+
+Basic Auth is defined in [RFC7617](https://tools.ietf.org/html/rfc7617).
+
+Install htpasswd command line tool, we assume Debian based
+system. Please refer the documentation of your Operating System or
+package management vendor how to install `htpasswd`:
+
+```
+% apt-get install apache2-utils
+```
+
+Create a htpasswd file `foo.passwd` and a use `catpain` with password `apassword`:
```

Comment:
"...and a use `catpain` with password..." => "...and use `captain` with password..."